src/lib/termdesc.c

```diff
@@ -802,6 +802,11 @@ int interrogate_terminfo(tinfo* ti, int fd, const char* termname, unsigned utf8,
         invertsixel = true;
       }
     }
+    if(qterm == TERMINAL_FOOT){
```

Comment:
`else if` probably makes more sense here, though it's not like the compiler can't figure this out for itself, so no worries.